crest/Assets/Crest/Crest/Scripts/Shapes/ShapeGerstner.cs

```diff
@@ -90,6 +90,22 @@ public partial class ShapeGerstner : MonoBehaviour, LodDataMgrAnimWaves.IShapeUp
 
         float _windSpeedWhenGenerated = -1f;
 
+        static int s_Count = 0;
```

Comment:
Wonder if s_InstanceCount or such is more descriptive

Reply:
Sure. Done.